recipes/Momentum/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "Momentum" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/facebookincubator/momentum/archive/v{{ version }}.tar.gz
+    sha256: f37bb8288b873de3b804c367eadeec81c02bff2ceff470f35a9427ed4b426bfe
+    patches:
+      - specify-output-dirs.patch
+      - include-cstdlib.patch
+
+build:
+  number: 0
+  skip: true  # [linux or osx]
```

Comment:
Is this broad skip intentional?

Reply:
Yes, I would like to enable them after it's converted to a feedstock repo for faster iteration, if that's acceptable :)

Reply:
If you enable them here, I would be able to help you a bit. It's surprising to see (like the first time) that someone got something to build on Windows but fails on the unixes.